### ISSUE TITLE
Bluetooth

### ISSUE THREAD

**Issue by banksio** [Hyrules/WinHue#251]
[Philips Hue bulbs now come with Bluetooth!](https://www.signify.com/global/our-company/news/press-releases/2019/20190627-signify-launches-philips-hue-with-bluetooth)

I'm not even sure there's an API or anything yet but I am up for experimenting with them (I have a couple) so could possibly add that functionality to WinHue at some stage? Would allow for more features than the official Bluetooth app supports.

Might be too tricky to build into WinHue, just creating this issue to track experiments for now...

**Comment by banksio** [comment]
By the way @Hyrules I've got a new laptop (Dell XPS 15) coming soon so once that arrives I'll be able to do some more work on the theming and hopefully finish it before the end of the summer.

**Comment by Hyrules** [comment]
Yes i'm aware that Philips Hue now support bluetooth. Thing is i'm not sure I want to implement bluetooth. I don't have any of those lights and I would buy LIFX over Philips Hue If I had to buy some lights again. LIFX are brighter, colors are better and bulb are independent. Bluetooth is very limited. I don't think this is a wise choice from philips. Range is also limited so people will have problem controling them across their house not an issue with the bridge. I Philips goes full on bluetooth I will most likely end  WinHue developpement and proceed to develop WinLIFX.

**Comment by banksio** [comment]
That's fair enough, the bulbs also have Zigbee still so they still add to the bridge etc. No pressure on adding the feature at all.

**Comment by mrquincle** [comment]
If there's enough information available about the bluetooth services/characteristics, I can see if our team can pick this up. It would make them hopefully controllable directly or through a Bluetooth mesh.
